Vodovoz/Filters/ViewModels/WarehouseJournalFilterViewModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using QS.Project.Filter;
+using QS.Project.Journal;
+using Vodovoz.Domain.Store;
+
+namespace Vodovoz.Filters.ViewModels
+{
+    public class WarehouseJournalFilterViewModel : FilterViewModelBase<WarehouseJournalFilterViewModel>, IJournalFilter
+    {
+        private bool restrictWithoutUnload;
+        public bool RestrictWithoutUnload
+        {
+            get => restrictWithoutUnload;
+            set => UpdateFilterField(ref restrictWithoutUnload, value, () => RestrictWithoutUnload);
```

Comment:
```suggestion
            set => UpdateFilterField(ref restrictWithoutUnload, value);
```
И так для всех, этот параметр лишний